src/core/AutoRest.Extensions/ParameterGroupExtensionHelper.cs

```diff
@@ -69,8 +69,8 @@ private static string GetParameterGroupName(string methodGroupName, string metho
                     string parameterGroupName;
                     if (specifiedGroupName == null)
                     {
-                        string postfix = extensionObject.Value<string>("postfix") ?? "Parameters";
-                        parameterGroupName = methodGroupName + "-" + methodName + "-" + postfix;
+                        string postfix = extensionObject.Value<string>("postfix").Capitalize() ?? "Parameters";
```

Comment:
`extensionObject.Value<string>("postfix")` may be null (that seems to be what makes CI fail)